src/test/java/uk/gov/hmcts/reform/iataskconfiguration/dmn/CamundaTaskInitiationTest.java

```diff
@@ -1644,6 +1644,23 @@ static Stream<Arguments> scenarioProvider() {
                     )
                 )
             ),
+            Arguments.of(
```

Comment:
I am missing a functional test for this new event
https://github.com/hmcts/ia-wa-post-deployment-ft-tests

Reply:
@MiguelMartinRojas Thomas suggested removing these changes completing since the task is already getting created by Manage Fee Update event. The requirement is just about the task auto completing so I will keep those